src/opers.c

```diff
@@ -6082,6 +6301,10 @@ static StructGVarFunc GVarFuncs [] = {
     GVAR_FUNC(CLEAR_HIDDEN_IMP_CACHE, 1, "flags"),
     GVAR_FUNC(WITH_HIDDEN_IMPS_FLAGS, 1, "flags"),
     GVAR_FUNC(InstallHiddenTrueMethod, 2, "filter, filters"),
+    GVAR_FUNC(RankFilter, 1, "filtorflags"),
+    GVAR_FUNC(CLEAR_IMP_CACHE, 0, ""),
+    GVAR_FUNC(WITH_IMPS_FLAGS, 1, "flags"),
+    GVAR_FUNC(WITH_IMPS_FLAGS_STAT, 0, ""),
```

Comment:
Please surround these four lines by `#if !defined(HPCGAP) ... #endif`, so that HPC-GAP won't be broken (as it will still use its version of `filter1.g`)